src/main.ts

```diff
@@ -0,0 +1,84 @@
+import { Actor } from 'apify';
+import { log } from 'crawlee';
+import { createServer } from 'http';
+
+import { addSearchRequest, createAndStartCrawlerPlaywright, createAndStartSearchCrawler } from './crawlers';
+import { UserInputError } from './errors.js';
+import { processInput } from './input';
+import { addTimeoutToAllResponses, sendResponseError } from './responses.js';
+import { CrawlerOptions, Input, ScraperSettings } from './types.js';
+import { parseParameters, checkForExtraParams, createRequestSearch } from './utils';
+
+await Actor.init();
+
+// Allow to run standby mode only when the actor is not running in the Apify platform (e.g. in local development)
+const RUN_STANDBY_MODE_AT_LOCAL = !Actor.isAtHome() && true;
+const TIMEOUT_MS = 60000;
+
+Actor.on('migrating', () => {
+    addTimeoutToAllResponses(60);
+});
+
+const server = createServer(async (req, res) => {
+    if (req.method !== 'HEAD') {
+        log.info(`Request received: ${req.method} ${req.url}`);
+    }
+    try {
+        const params = parseParameters(req.url!);
+        checkForExtraParams(params);
+        log.info(`Received input parameters: ${JSON.stringify(params)}`);
+        const { input } = await processInput(params as Partial<Input>);
+
+        const crawlerRequest = createRequestSearch(input.queries, input.maxResults);
+        const crawlerOptions: CrawlerOptions = {
+            proxyConfigurationOptions: { groups: [input.proxyTypeSearchCrawler] },
+        };
+
+        setTimeout(() => {
+            const timeoutErrorMessage = {
+                errorMessage: `Response timed out.`,
+            };
+            sendResponseError(crawlerRequest.uniqueKey!, JSON.stringify(timeoutErrorMessage));
+        }, TIMEOUT_MS);
+
+        await addSearchRequest(crawlerRequest, res, crawlerOptions, input.maxResults);
+    } catch (e) {
+        const error = e as Error;
+        const errorMessage = { errorMessage: error.message };
+        const statusCode = error instanceof UserInputError ? 400 : 500;
+        res.writeHead(statusCode, { 'Content-Type': 'application/json' });
+        res.end(JSON.stringify(errorMessage));
+    }
+});
+
+if ((Actor.isAtHome() && Actor.getEnv().metaOrigin === 'STANDBY') || RUN_STANDBY_MODE_AT_LOCAL) {
```

Comment:
You don't need (Actor.isAtHome() and RUN_STANDBY_MODE_AT_LOCAL, you can pass it env var locally when running

```
APIFY_META_ORIGIN=STANDBY apify run
```

Reply:
oh, ok. Thanks!